app/src/main/java/com/goryachok/forecastapp/data/RemoteDataSource.kt

```diff
@@ -0,0 +1,80 @@
+package com.goryachok.forecastapp.data
+
+import android.util.Log
+import com.goryachok.forecastapp.api.OpenWeatherMapAPI
+import com.goryachok.forecastapp.base.RemoteEntity
+import com.goryachok.forecastapp.model.domain.ForecastEntity
+import com.goryachok.forecastapp.model.domain.WeatherEntity
+import javax.inject.Inject
+import kotlin.reflect.KClass
+
+class RemoteDataSource @Inject constructor(val api: OpenWeatherMapAPI) {
+
+    companion object {
+        private const val DAGGER_TAG ="DaggerDebug"
+
+        private const val FORECAST_QUERY_PARAM = "weather"
+        private const val WEATHER_QUERY_PARAM = "forecast"
+    }
+
+    init {
+        Log.d(DAGGER_TAG, javaClass.name)
+    }
+
+//    fun getWeatherData(
```

Comment:
Please either add "TODO" if you're planning to use commented code, or remove unused commented code.

Reply:
Removed